client/components/focusable/index.jsx

```diff
@@ -30,17 +27,15 @@ class Focusable extends Component {
 	};
```

Comment:
Should the above be an `if/else if`? Does it make sense to handle a keyPress via `onClick` _and_ `onKeyDown`?

Reply:
Maybe it should be `else if`, but then it might be unclear that `onKeyDown` is _not_ called on <kbd>Enter</kbd> & <kbd>Space</kbd>.